mongorestore/testdata/tools-1856/js/d5.js

```diff
@@ -0,0 +1,28 @@
+// Dataset 8: Deeply nested arrays (100 layers of nesting)
```

Comment:
```suggestion
// Dataset 5: Deeply nested arrays (100 layers of nesting)
```
[nit] the comment doesn't match the filename